src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -332,21 +331,4 @@ private UserTenantEntity toEntity(UserTenant userTenantDto, UUID consortiumId, U
     entity.setIsPrimary(IS_PRIMARY_FALSE);
     return entity;
   }
-
-  private PermissionUser createPermissionUser(String userId) {
-    List<String> emptyPermissionList = new ArrayList<>();
-    Optional<PermissionUser> permissionUserOptional = permissionsClient.get("userId==" + userId)
-      .getPermissionUsers()
-      .stream()
-      .findFirst();
-    if (permissionUserOptional.isPresent()) {
-      // this case possible because for initial admin users setup we are creating user and permissions separately
```

Comment:
decided to delete this check if permission user already created to simplify things,
because this case when permission user created without shadow admin user is not possible now